DEHPMatlab/ViewModel/NetChangePreview/HubNetChangePreviewViewModel.cs

```diff
@@ -621,7 +621,7 @@ private void UpdateTreeBasedOnSelection(IEnumerable<MatlabWorkspaceRowViewModel>
 
                 var elementRow = this.VerifyElementIsInTheTree(elementBase.Value.First());
 
-                if (elementClone is ElementDefinition elementDefition)
+                if (elementClone is ElementDefinition { Container: { } } elementDefition)
```

Comment:
Why do you have to check whether the container is set here?

Reply:
If there is no Top Element, the UpdateChild throws an Exception because the Container of the ElementDefinition is null